mappings/net/minecraft/server/ServerCommandHandler.mapping

```diff
@@ -0,0 +1,19 @@
+CLASS net/minecraft/class_426 net/minecraft/server/ServerCommandHandler
+	FIELD field_1686 logger Ljava/util/logging/Logger;
+	FIELD field_1687 minecraftServer Lnet/minecraft/server/MinecraftServer;
+	METHOD <init> (Lnet/minecraft/server/MinecraftServer;)V
+		ARG 1 minecraftServer
+	METHOD method_1411 ExecuteCommand (Lnet/minecraft/class_38;)V
```

Comment:
Should this be `executeCommand` instead to have consistent capitalization?

Reply:
Yes, thank you for catching that. 10 years of C# conventions are messing with my head here 😂